mission.md

```diff
@@ -0,0 +1,17 @@
+# Mission
+
+We are on a mission to change the way that analytics is done. We believe that [analysts should work more like software engineers](https://medium.com/fishtown-analytics/building-a-mature-analytics-workflow-the-analyst-collective-viewpoint-7653473ef05b), and we spend our time thinking about, implementing, building tools for, and training analysts on this workflow. This mission animates everything about Fishtown Analytics: the way we work, the work we choose to do, the clients we work with, and the people we hire.
+
+We care about this mission for a very simple reason: we want to help analysts be more effective. We’re analysts ourselves, and have personally witnessed a dramatic increase in our productivity by implementing this workflow. We want to help others do the same.
```

Comment:
I think we need another sentence here indicating why we, or anybody else, should care if analysts are more effective. Why is analytics a worthwhile pursuit? Why is improving the way analytics is done valuable? I don't think it's quite enough to say "we're analysts, so we care about this." Sort of begs the question IMO

I have thoughts about this, but I am unsure if they constitute Corp Mission or Drew's Search for Meaning in a Cold, Cold Universe. Also willing to accept "Analytics is important" as an axiom if this discussion is too far removed from the purpose of this document.